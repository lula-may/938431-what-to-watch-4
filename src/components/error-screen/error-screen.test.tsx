import * as React from "react";
import * as renderer from "react-test-renderer";
import ErrorScreen from "./error-screen";

describe(`ErrorComponent`, () => {
  it(`should render correctly ErrorScreen component`, () => {
    const tree = renderer.create(
        <ErrorScreen/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
