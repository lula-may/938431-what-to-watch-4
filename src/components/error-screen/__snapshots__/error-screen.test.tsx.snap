// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorComponent should render correctly ErrorScreen component 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href=""
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    >
      Loading Error
    </h1>
  </header>
  <div
    className="sign-in user-page__content"
  >
    <div
      className="sign-in__message"
    >
      <p>
        Sorry, something went wrong.
      </p>
      <p>
         Please try again later.
      </p>
    </div>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href=""
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
